### ISSUE TITLE
Broken layout and ether input in v3.0.9

### ISSUE THREAD

**Issue by dvinubius** [scaffold-eth/eth-components#37]
When running the default scaffold-eth/typescript app, which uses eth-components v3.0.9

- some of the layout is misaligned
- the **ether input** has a faulty check and results in **not displaying the switch**

![Screenshot 2022-01-18 at 13 43 22](https://user-images.githubusercontent.com/32189942/149932232-cc59fc65-96a2-4093-8e30-aee3f1d65f6c.png)
![Screenshot 2022-01-18 at 13 43 04](https://user-images.githubusercontent.com/32189942/149932240-2f087ea5-aced-44f2-80c7-c7af81ac2255.png)
![Screenshot 2022-01-18 at 13 42 53](https://user-images.githubusercontent.com/32189942/149932248-41ee90c8-69c6-4ff0-9bf6-f9f522d96b57.png)

I have done the changes in my factory setup but I guess most users would have to do them also.

The result

![Screenshot 2022-01-18 at 13 54 15](https://user-images.githubusercontent.com/32189942/149933003-12d018b7-a33d-4e2e-9a8b-7edf424ccc67.png)
![Screenshot 2022-01-18 at 13 54 36](https://user-images.githubusercontent.com/32189942/149933009-a044c2d1-148f-46b0-85c4-66530782efd4.png)
![Screenshot 2022-01-18 at 13 54 45](https://user-images.githubusercontent.com/32189942/149933016-88aa1cc4-1a5a-48ef-b252-7cdc991e2ba8.png)

⚠️ I don't mean the changes in **background color or font**, but strictly **the above mentioned layout issues**

I could make the fixes myself and PR

**Comment by ShravanSunder** [comment]
@dvinubius yes please!

**Comment by dvinubius** [comment]
@ShravanSunder made the changes but the layout fix in the `<Address/> `only works with an explicit css import in scaffold eth. 

It may be possible to hack a fix for the misaligned copy icon without css, to keep eth-components more plug-and-play, but I found no quick way to do it.

Since we're planning to redesign eth-components anyway I figured it's not worth investing more effort in this now.

The corresponding PR in scaffold-eth is also opened: https://github.com/scaffold-eth/scaffold-eth-typescript/pull/69
It should pass the tests once this present PR is merged.
